test/test_wiki_trac_rst_convert.py

```diff
@@ -36,6 +46,41 @@ def test_newline(self):
         """
         self.assertConvertedContent('', '\n')
 
+    def test_single_content_directive(self):
+        """
+        Ensure only one RST content directive, not multiple.
+        """
+
+        self.assertConvertedContent(
+            'Sample content',
+
+            '.. contents::\n'
+            '\n'
+            'Sample content'
+        )
+
+        self.assertConvertedContent(
```

Comment:
There is an extra space - I found some files where there are 2 spaces instead of 1 between the `..` and the `contents::`. I will split the test case to clarify.